resources/themes/cura-light/images/first_run_Ultimaker_cloud.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Lots of unused IDs in this file.

Reply:
I'm going to leave the SVGs as they are for now.

Reply:
Removed in f4cef01c7